shopinvader_validation_token/data/mail_template.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo noupdate="1">
+
+    <record id="mail_template_security_token" model="mail.template" forcecreate="True">
+        <field name="name">Shopinvader - Security token</field>
+        <field
+            name="email_from"
+        >${(object.shopinvader_backend_id.company_id.email or '')|safe}</field>
+        <field name="subject">Security token</field>
+        <field name="email_to">${object.email|safe}</field>
+        <field name="model_id" ref="model_shopinvader_security_token" />
+        <field name="auto_delete" eval="False" />
+        <field name="lang">${object.lang}</field>
+        <field
+            name="body_html"
+        ><![CDATA[ Your token: ${object.token} valid until ${format_tz(object.expiration_datetime)} ]]></field>
```

Comment:
format_tz is not available on Odoo 14.0:

```
2022-06-15 08:47:27,111 1 ERROR devel werkzeug: Error on request:
Traceback (most recent call last):
  File "/opt/odoo/custom/src/odoo/odoo/addons/base/models/ir_http.py", line 237, in _dispatch
    result = request.dispatch()
  File "/opt/odoo/custom/src/odoo/odoo/http.py", line 806, in dispatch
    r = self._call_function(**self.params)
  File "/opt/odoo/custom/src/odoo/odoo/http.py", line 359, in _call_function
    return checked_call(self.db, *args, **kwargs)
  File "/opt/odoo/custom/src/odoo/odoo/service/model.py", line 94, in wrapper
    return f(dbname, *args, **kwargs)
  File "/opt/odoo/custom/src/odoo/odoo/http.py", line 347, in checked_call
    result = self.endpoint(*a, **kw)
  File "/opt/odoo/custom/src/odoo/odoo/http.py", line 912, in __call__
    return self.method(*args, **kw)
  File "/opt/odoo/custom/src/odoo/odoo/http.py", line 531, in response_wrap
    response = f(*args, **kw)
  File "/opt/odoo/custom/src/queue/queue_job/controllers/main.py", line 107, in runjob
    self._try_perform_job(env, job)
  File "/opt/odoo/custom/src/queue/queue_job/controllers/main.py", line 37, in _try_perform_job
    job.perform()
  File "/opt/odoo/custom/src/queue/queue_job/job.py", line 520, in perform
    self.result = self.func(*tuple(self.args), **self.kwargs)
  File "/opt/odoo/custom/src/odoo-shopinvader/shopinvader/models/shopinvader_notification.py", line 85, in send
    self.sudo()
  File "/opt/odoo/custom/src/odoo/addons/mail/models/mail_template.py", line 262, in send_mail
    values = self.generate_email(res_id, ['subject', 'body_html', 'email_from', 'email_to', 'partner_to', 'email_cc', 'reply_to', 'scheduled_date'])
  File "/opt/odoo/custom/src/odoo/addons/account_edi/models/mail_template.py", line 21, in generate_email
    res = super().generate_email(res_ids, fields)
  File "/opt/odoo/custom/src/odoo/addons/mail/models/mail_template.py", line 182, in generate_email
    generated_field_values = template._render_field(
  File "/opt/odoo/custom/src/odoo/addons/mail/models/mail_render_mixin.py", line 475, in _render_field
    return dict(
  File "/opt/odoo/custom/src/odoo/addons/mail/models/mail_render_mixin.py", line 478, in <genexpr>
    for res_id, rendered in template._render_template(
  File "/opt/odoo/custom/src/odoo/addons/mail/models/mail_render_mixin.py", line 398, in _render_template
    rendered = self._render_template_jinja(template_src, model, res_ids, add_context=add_context)
  File "/opt/odoo/custom/src/odoo/addons/mail/models/mail_render_mixin.py", line 355, in _render_template_jinja
    raise UserError(_("Failed to render template : %s", e))
Exception

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/werkzeug/serving.py", line 306, in run_wsgi
    execute(self.server.app)
  File "/usr/local/lib/python3.8/site-packages/werkzeug/serving.py", line 294, in execute
    application_iter = app(environ, start_response)
  File "/opt/odoo/custom/src/odoo/odoo/service/server.py", line 441, in app
    return self.app(e, s)
  File "/opt/odoo/custom/src/odoo/odoo/service/wsgi_server.py", line 113, in application
    return application_unproxied(environ, start_response)
  File "/opt/odoo/custom/src/odoo/odoo/service/wsgi_server.py", line 88, in application_unproxied
    result = odoo.http.root(environ, start_response)
  File "/opt/odoo/custom/src/odoo/odoo/http.py", line 1328, in __call__
    return self.dispatch(environ, start_response)
  File "/opt/odoo/custom/src/odoo/odoo/http.py", line 1294, in __call__
    return self.app(environ, start_wrapped)
  File "/usr/local/lib/python3.8/site-packages/werkzeug/middleware/shared_data.py", line 220, in __call__
    return self.app(environ, start_response)
  File "/opt/odoo/custom/src/odoo/odoo/http.py", line 1518, in dispatch
    result = ir_http._dispatch()
  File "/opt/odoo/custom/src/odoo/addons/auth_signup/models/ir_http.py", line 19, in _dispatch
    return super(Http, cls)._dispatch()
  File "/opt/odoo/custom/src/odoo/addons/web_editor/models/ir_http.py", line 21, in _dispatch
    return super(IrHttp, cls)._dispatch()
  File "/opt/odoo/custom/src/odoo/addons/utm/models/ir_http.py", line 29, in _dispatch
    response = super(IrHttp, cls)._dispatch()
  File "/opt/odoo/custom/src/odoo/addons/http_routing/models/ir_http.py", line 512, in _dispatch
    result = super(IrHttp, cls)._dispatch()
  File "/opt/odoo/custom/src/odoo/odoo/addons/base/models/ir_http.py", line 241, in _dispatch
    return cls._handle_exception(e)
  File "/opt/odoo/custom/src/odoo/addons/utm/models/ir_http.py", line 34, in _handle_exception
    response = super(IrHttp, cls)._handle_exception(exc)
  File "/opt/odoo/custom/src/odoo/addons/http_routing/models/ir_http.py", line 602, in _handle_exception
    return super(IrHttp, cls)._handle_exception(exception)
  File "/opt/odoo/custom/src/odoo/odoo/addons/base/models/ir_http.py", line 209, in _handle_exception
    return request._handle_exception(exception)
  File "/opt/odoo/custom/src/odoo/odoo/http.py", line 744, in _handle_exception
    return super(HttpRequest, self)._handle_exception(exception)
  File "/opt/odoo/custom/src/odoo/odoo/http.py", line 315, in _handle_exception
    raise exception.with_traceback(None) from new_cause
odoo.exceptions.UserError: Failed to render template : 'format_tz' is undefined - - -
```

Reply:
@HviorForgeFlow Thanks for your review. I just fix it.
The `format_tz` is no more available in `v14.0`